### DISCUSSION TITLE
Help on detecting the bulbs

### DISCUSSION THREAD

**Discussion by dncsmith** [sandarshsridhar/wiz-spotify-connected#40] [Q&A]
I would like some help getting the lights to be detected by your app, please. It detects the music I'm playing in my WSL terminal, but the lights stay lit to what they're normally set at. If I visit http://localhost:8888/rooms, it returns an empty JSON object, which raises my suspicion that the lights were not detected.

In the Wiz app, I already had "Allow local communication" enabled. I am using version 2 of the app, if that could be a reason?

In the .env file, what IP address do I enter for the BROADCAST_ADDRESS variable? Is it the router's IP address? I did remove the variable entirely, but to no avail.

Any help will be greatly appreciated!

Thank you! 😊

**Comment by sandarshsridhar** [comment]
Hi, yes, you could try using your router's broadcast address. It's not the router's IP address. Instead, it would be something like this. If your local devices have IPs like 192.168.1.15, then the broadcast address would be 192.168.1.255. You could perhaps use this link to figure out the address: https://www.wikihow.com/Calculate-Network-and-Broadcast-Address. Also, I have got my stuff to work with 255.255.255.255 and 0.0.0.0 as well. It really depends on your router and the network setup. I hope this helps :)
